packages/vue-components/src/SiteNavButton.vue

```diff
@@ -51,15 +57,59 @@ export default {
 </script>
 
 <style scoped>
+
     .toggle-site-nav-button {
         cursor: pointer;
+        height: 53px;
         left: 0;
         padding: 15px;
         position: absolute;
+        transform: all rotate(0deg);
+        transition: all 0.25s ease-in-out;
+        width: 52px;
+    }
+
+    .toggle-site-nav-button span {
+        background-color: #000;
+        display: block;
+        height: 3px;
+        left: 15px;
+        position: absolute;
+        top: 50%;
+        transform: all rotate(0deg);
+        transition: all 0.25s ease-in-out;
+        width: 21px;
+    }
+
+    .toggle-site-nav-button > span:nth-child(1) {
+        transform: translateY(-6px);
```

Comment:
minor spacing nit: (likewise for the reverse `translateY`)
```suggestion
        transform: translateY(-8px);
```